wiki/Ranking_Criteria/zh.md

```diff
@@ -1,22 +1,19 @@
----
-outdated: true
-outdated_since: 2da3d3ec26261552cc7ae67298d375b939addbf3
----
-
 # Ranking Criteria
 
 *查看每个游戏模式具体的标准，请参阅 [osu!](/wiki/rc_osu!)，[osu!taiko](/wiki/rc_osu!taiko)，[osu!catch](/wiki/rc_osu!catch) 和 [osu!mania](/wiki/rc_osu!mania)。*
 
-这里会列出[官方论坛](https://osu.ppy.sh/community/forums/87)中的新增内容。按照惯例，任何正在讨论的规定都将列在此处，一旦讨论达成一致，就会更新它们。
+**ranking criteria** 列出了让[谱面](/wiki/Beatmaps)通过[谱面ranking程序](/wiki/Beatmap_ranking_procedure)必须遵守的[规则和准则](#general-terms)
```

Comment:
大小写；在ranking criteria用原文的情况下个人认为应当视作专有名词

EN: Capitalization issues. Personal opinion is that "ranking criteria" is used as is so should use proper noun capitalization (instead of what is in the original text)

```suggestion
**Ranking Criteria** 列出了让[谱面](/wiki/Beatmaps)通过[谱面ranking程序](/wiki/Beatmap_ranking_procedure)必须遵守的[规则和准则](#general-terms)
```

Reply:
@TicClick Are you OK with inconsistent capitalization with original?

Reply:
that isn't a proper noun, the only reason I see for capitalization here is the beginning of a sentence. if it's used as such as a loan word and there is no short equivalent of a phrase "a set of rules that all ranked beatmaps must follow", however, it's fine